homeassistant/components/prometheus/__init__.py

```diff
@@ -807,6 +817,34 @@ def _handle_update(self, state: State) -> None:
         value = self.state_as_number(state)
         metric.labels(**self._labels(state)).set(value)
 
+    def _handle_alarm_control_panel(self, state: State) -> None:
+        current_mode = state.state
```

Comment:
The `alarm_control_panel` refers to these as "states" rather than "modes", so I'd suggest to use the work "mode" here and below.

Reply:
Changed.